doc/api/modules.md

```diff
@@ -542,6 +542,7 @@ A reference to the current module, see the section about the
 [`module` object][]. In particular, `module.exports` is used for defining what
 a module exports and makes available through `require()`.
 
+<a id="require"></a>
```

Comment:
This isn't necessary as the html generator for the markdown will create these id attributes automatically.